apps/test/unit/lib/ui/SettingsCogTest.js

```diff
@@ -157,19 +152,14 @@ describe('SettingsCog', () => {
         expect(wrapper).to.be.blank;
       });
 
-      it('calls handleToggleMaker when clicked', () => {
+      it('asks for confirmation when clicked', () => {
```

Comment:
This one wasn't testing anything in the react component, so I changed it to be a more useful test.